test/Integration/openmp/tpp-run-parallel.mlir

```diff
@@ -1,6 +1,8 @@
-// RUN: tpp-run %s -def-parallel=0 -e entry -entry-point-result=void -print-mlir=late -print 2>&1 | \
+// RUN: tpp-opt %s -default-tpp-passes | \
```

Comment:
Why do you need to pipe `tpp-opt`?

Reply:
We need to make sure we get from scf.forall to scf.parallel. Previously we were only lowering: `scf.parallel` using tpp-run this did not catch the regression.

Reply:
That's what I want to understand. both `tpp-opt` and `tpp-run` need to converge to the same IR, especially when both are running the default TPP passes. If they don't converge, then there is a pipeline problem.

Reply:
there was not need for `tpp-opt`.